vision/vae_mnist/README.md

```diff
@@ -1,6 +1,15 @@
-# Variational Autoencoder(VAE)
+# Variational Autoencoder (VAE)
+
+<img src="..\vae_mnist\docs\vae.png" width="500">
+
+## Model Info
+
+Variational Autoencoder ( VAE ) came into existence in 2013, when Diederik et al. published a paper [Auto-Encoding Variational Bayes](https://arxiv.org/pdf/1312.6114.pdf). This paper was an extension of the original idea of Auto-Encoder primarily to learn the useful distribution of the data. Variational Autoencoder was inspired by the methods of the variational bayesian and graphical model. VAE is rooted in Bayesian inference, i.e., it wants to model the underlying probability distribution of data to sample new data from that distribution.
+
+VAE has one fundamentally unique property that separates them from vanilla autoencoder, and it is this property that makes them so useful for generative modeling: their latent spaces are, by design, continuous, allowing easy random sampling and interpolation.
```

Comment:
also autoencoders have a continuous latent space, the difference is that the autoencoders are not  probabilistic models.